src/leap/mail/incoming/service.py

```diff
@@ -393,9 +393,7 @@ def _process_decrypted_doc(self, doc, data):
             defer.returnValue(False)
 
         # ok, this is an incoming message
-        rawmsg = msg.get(self.CONTENT_KEY, None)
-        if not rawmsg:
-            return ""
+        rawmsg = msg.get(self.CONTENT_KEY, "")
```

Comment:
Should I revert this change? Just realized that it was returning the empty string straight forward instead of passing to the _maybe_decrypt_msg function.